frontend/database/modules/text_classification.js

```diff
@@ -16,14 +16,19 @@
  */
 
 import { ObservationDataset } from "@/models/Dataset";
+import { TextClassificationDataset } from "@/models/TextClassification";
 
 const getters = {};
 
 const actions = {
-  async setLabels(_, { dataset, labels }) {
-    return await ObservationDataset.dispatch("setUserData", {
-      dataset,
-      data: { labels },
+  async onSaveTokenDatasetSettings(context, { datasetId, newLabel }) {
```

Comment:
This function name references token but is used in Text Classifier

Reply:
I did a refacto in the PR for the label management component: 
- to remove these actions from token_dataset.js and text_classification.js 
- to use directly the "onAddNewLabels" action from dataset.js instead